build/src/main.rs

```diff
@@ -79,17 +86,48 @@ fn main() {
 			.help("Injects RUNTIME_VERSION global export")
 			.takes_value(true)
 			.long("runtime-version"))
+		.arg(Arg::with_name("source_target")
+			.help("Cargo target type kind (wasm32-unknown-unknown/emscripten)")
+			.takes_value(true)
+			.long("target"))
+		.arg(Arg::with_name("final_name")
+			.help("Final wasm binary name")
+			.takes_value(true)
+			.long("final"))
+		.arg(Arg::with_name("save_raw")
+			.help("Save intermediate raw bytecode to path")
+			.takes_value(true)
+			.long("save-raw"))
 		.get_matches();
 
     let target_dir = matches.value_of("target").expect("is required; qed");
     let wasm_binary = matches.value_of("wasm").expect("is required; qed");
+	let mut source_input = source::SourceInput::new(target_dir, wasm_binary);
 
-	process_output(target_dir, wasm_binary).expect("Failed to process cargo target directory");
+	let source_target_val = matches.value_of("source_target").unwrap_or_else(|| source::EMSCRIPTEN_PATH);
+	if source_target_val == source::UNKNOWN_PATH {
```

Comment:
`UNKNOWN_PATH` looks sort of confusing to me. 

Maybe we can come up with a better name? 
maybe `s/UNKNOWN_/LLVM_WASM_` ?

Reply:
there are two targets, `wasm32-unknown-unknown` and `wasm32-unknown-emscripten`, with only last word changed, so distinguish by this word makes sense to me

there is no strict relationship between `unknown` and `llvm` whatsoever

Reply:
indeed,

My concern is that `UNKNOWN_PATH` is read as something like, well, unknown path, and not as desired "path to a dir where cargo would put binaries for a `wasm32-unknown-unknown` target".

However, I don't want to block the merge on this, since scope of this identifiers is quite narrow and in full context these identifiers can make sense for a reader.

Reply:
aye, renamed to UNKNOWN_TRIPLET